scripts/ansible/client-auto-update-playbook.yml

```diff
@@ -0,0 +1,65 @@
+- hosts: zeitgeist
+  gather_facts: no
+  become: true 
+  environment: 
+    RELEASE_VERSION: "{{ lookup('env', 'RELEASE_VERSION') }}"
```

Comment:
Where do we set `RELEASE_VERSION`? In the environment variables? What happens, if it is not set? Can we print the RELEASE_VERSION in the Github Actions Debug Console?